package-lock.json

```diff
@@ -4,74 +4,402 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
```

Comment:
How come we've got so many changes in here?

Reply:
I'm not sure - my assumption was that because this repo doesn't get added to very often, that an `npm install` hadn't been committed in a while and that's what these changes are from

Reply:
Looks like stylelint just has a hella lot of dependencies!